debezium-ddl-parser/src/test/resources/mysql/examples/ddl_alter.sql

```diff
@@ -2,10 +2,12 @@
 -- Alter Table
 alter table ship_class add column ship_spec varchar(150) first, add somecol int after start_build, algorithm=instant;
 alter table t3 add column (c2 decimal(10, 2) comment 'comment`' null, c3 enum('abc', 'cba', 'aaa')), add index t3_i1 using btree (c2) comment 'some index';
+alter table t3 add column (c4 decimal(10, 2) comment 'comment`' null, add index t3_i2 using btree (c4) comment 'some index';
 alter table t2 add constraint t2_pk_constraint primary key (1c), alter column `_` set default 1;
 alter table ship_class change column somecol col_for_del tinyint first;
 alter table ship_class drop col_for_del;
 alter table t3 drop index t3_i1;
+alter table t3 drop index t3_i2 if exists;
```

Comment:
Thanks for the PR!
Is this example correct? It seems to me that accroding to https://mariadb.com/kb/en/drop-index/  the `ON <table>` part is mandatory.

Reply:
updated PR with drop index if exists _indexname_

ON is not mandatory when used from alter table.  your syntax reference is about drop index DDL.